tox.ini

```diff
@@ -3,7 +3,7 @@ isolated_build = true
 envlist = py38, py39
 
 [testenv]
-whitelist_externals = poetry
+allowlist_externals = poetry
```

Comment:
This change isn't actually necessary, but don't think it does any harm: we specify `tox = "^3.27.1"` in pyproject.toml which supports both `whitelist_externals` and `allowlist_externals`; v4.x only supports `allowlist_externals`.

_(I ended up with this after getting confused about whether Poetry was supposed to install Tox or vice versa, so I installed Tox v4 globally, and with that the use of `whitelist_externals` threw an error)_